packages/jest-changed-files/src/hg.js

```diff
@@ -45,12 +45,22 @@ const adapter: SCMAdapter = {
     }
     args.push(...includePaths);
 
-    const result = await execa('hg', args, {cwd, env});
+    try {
+      const result = await execa('hg', args, {cwd, env});
+
+      return result.stdout
+        .split('\n')
+        .filter(s => s !== '')
+        .map(changedPath => path.resolve(cwd, changedPath));
+    } catch (e) {
+      const errorMsg = e.message.split('\n')[1];
```

Comment:
Instead of doing this, you should use `formatExecError` from https://github.com/facebook/jest/blob/8e6b34520517033be0cbd866ccd70b91cb237696/packages/jest-message-util/src/index.js#L102